### ISSUE TITLE
List should be treated as invalid unpacking targets

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#1256]
# Rule request

## Thesis

Currently, this code passes our linter:

```python
[first, second] = some()
```

But, it is wrong. We need to force:

```python
first, second = some()
```

So, we need to disallow `ast.List` as an unpacking target.

## Reasoning

There's an already a working way of doing unpacking assignment: tuples.
We don't need one more way to do it.

I would say this is a consistency rule.

**Comment by skarzi** [comment]
Can I prepare PR for this issue?

**Comment by sobolevn** [comment]
@skarzi sure! Thanks a lot!
